kimchi/src/proof.rs

```diff
@@ -44,29 +56,20 @@ pub struct LookupEvaluations<Field> {
 /// - **Non chunked evaluations** `Field` is instantiated with a field, so they are single-sized#[serde_as]
 #[serde_as]
 #[derive(Clone, Serialize, Deserialize)]
-#[serde(bound(
-    serialize = "Vec<o1_utils::serialization::SerdeAs>: serde_with::SerializeAs<Field>",
-    deserialize = "Vec<o1_utils::serialization::SerdeAs>: serde_with::DeserializeAs<'de, Field>"
-))]
-pub struct ProofEvaluations<Field> {
+pub struct ProofEvaluations<Eval> {
     /// witness polynomials
-    #[serde_as(as = "[Vec<o1_utils::serialization::SerdeAs>; COLUMNS]")]
-    pub w: [Field; COLUMNS],
+    pub w: [Eval; COLUMNS],
```

Comment:
why not just have `PointEvaluations` instead of `Eval`?

Reply:
So that, later, we can use this for
* the polynomials
* the commitments
* the evaluations
rather than having a separate structure for each. Specialising it now just means that I'll have to un-specialise it later.

Reply:
aren't these also going to use `PointEvaluations` since it is generic as well now?

Reply:
The commitments and the polynomials don't have anything to do with `zeta` or `zeta_omega`, they're computed before we know what they are. Feels very odd to try to shoehorn them into this type.

Reply:
when you say later you mean in the other PRs in this series? Otherwise I think this is premature optimization